app/models/host/managed.rb

```diff
@@ -519,46 +519,6 @@ def setBuild
     errors.empty?
   end
 
-  # this method accepts a puppets external node yaml output and generate a node in our setup
-  # it is assumed that you already have the node (e.g. imported by one of the rack tasks)
-  def importNode(nodeinfo)
```

Comment:
I think this commit should also drop the only caller (https://github.com/theforeman/foreman/blob/0e7b3dd50bb930a0500ddc834146aecbb527a837/lib/tasks/puppet.rake#L27-L58)